test/e2e/test_cluster_configs/sgx.json

```diff
@@ -0,0 +1,46 @@
+{
+	"env": {
+		"REGION_OPTIONS": "westeurope"
+	},
+	"apiModel": {
+        "apiVersion": "vlabs",
+        "properties": {
+          "orchestratorProfile": {
+            "orchestratorType": "Kubernetes",
+            "orchestratorRelease": "1.13",
+            "kubernetesConfig": {
+              "useManagedIdentity": true
```

Comment:
Can we skip this part of the test? If not, more will be necessary to make MSI work w/ our E2E test runner. Here's the test output:

```
Error from deployment for kubernetes-westeurope-76947 in resource group kubernetes-westeurope-76947:exit status 1
 2019/10/04 00:16:00 Command Output: ERROR: Azure Error: InvalidTemplateDeployment
 Message: Deployment failed with multiple errors: 'Authorization failed for template resource '<resource>' of type 'Microsoft.Authorization/roleAssignments'. The client '<client>' with object id '<object>' does not have permission to perform action 'Microsoft.Authorization/roleAssignments/write' at scope '/subscriptions/<sub-id>/resourceGroups/kubernetes-westeurope-76947/providers/Microsoft.Authorization/roleAssignments/<role-assignment>'.:Authorization failed for template resource '<resource>' of type 'Microsoft.Authorization/roleAssignments'. The client '<client>' with object id '<object>' does not have permission to perform action 'Microsoft.Authorization/roleAssignments/write' at scope '/subscriptions/<sub-id>/resourceGroups/kubernetes-westeurope-76947/providers/Microsoft.Authorization/roleAssignments/<role-assignment>.'
```

Reply:
Sure, I dont think its essential for the e2e test.